bin/update_i18n.rb

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../lib", __dir__)
+
+require 'bundler/setup'
+require 'manageiq/release'
+require 'optimist'
+
+opts = Optimist.options do
+  opt :branch,  "The target branch",:type => :string,  :required => false
```

Comment:
When you say branch here do you literally mean a "branch" or is it a repo_set? (Also a couple of space-after-comma issues here).